packages/graphback-cli/src/helpers/db.ts

```diff
@@ -6,7 +6,7 @@ import { DatabaseSchemaManager, GraphQLBackendCreator } from 'graphback';
 import { logError, logInfo } from '../utils'
 import { checkDirectory } from './common'
 
-const configPath = `${process.cwd()}/config.json`
+const configPath = `${process.cwd()}/.graphback`
```

Comment:
Just `.graphback` or `.graphback.json`

Reply:
@ankitjena hey, 
We have just realized that for now it needs to stay as .json, changed it graphback.json